src/routes/docs/tutorials/react/step-4/+page.markdoc

```diff
@@ -29,6 +29,7 @@ export function UserProvider(props) {
   async function login(email, password) {
     const loggedIn = await account.createEmailPasswordSession(email, password);
     setUser(loggedIn);
+    navigateTo("/");
```

Comment:
Will this work considering it's not imported and we don't mention any library or anything for the routing?